tests/testmultistream.nim

```diff
@@ -278,8 +278,8 @@ suite "Multistream select":
       await transport1.close()
 
       await allFuturesThrowing(
-        handlerWait1.wait(5000.millis),
-        handlerWait2.wait(5000.millis))
+        handlerWait1.wait(30.seconds),
```

Comment:
Meh, i think that test has a real issue that needs investigating, increasing the time will most likely hide it.

Reply:
mac is likely running CI as a nested VM, I think it's really timing, cos giving it time it won't leak.

Reply:
There is a real `switch.stop` issue btw still , the one that still fails, I will look at that first.